content/applications/inventory_and_mrp/inventory/product_management/product_replenishment/reordering_rules.rst

```diff
@@ -79,43 +83,187 @@ Reordering Rules`, then click :guilabel:`Create`, and fill out the new line as f
 
 .. tip::
    Reordering rules can also be created from each product form. To do so, navigate to
-   :menuselection:`Inventory --> Products --> Products`, then select a product. Click on
-   :menuselection:`Reordering Rules --> Create`, then fill out the new line as detailed above.
+   :menuselection:`Inventory app --> Products --> Products`, and select a product. Then, click the
+   :guilabel:`Reordering Rules` smart button, and click :guilabel:`New` to fill out the new line, as
+   detailed above.
+
+For advanced usage of reordering rules, learn about the following reordering rule fields:
+
+#. :ref:`Route <inventory/product_management/route>`
+#. :ref:`Trigger <inventory/product_management/trigger>`
+#. :ref:`Visibility Days <inventory/product_management/visibility-days>`
```

Comment:
- bullet list here since order doesn't matter
- remove capital letters for words that are not proper nouns
- order the anchors in order of how the content appears

```suggestion
- :ref:`Trigger <inventory/product_management/trigger>`
- :ref:`Visibility days <inventory/product_management/visibility-days>`
- :ref:`Route <inventory/product_management/route>`
```